zubhub_backend/zubhub/notifications/models.py

```diff
@@ -21,6 +22,7 @@ class Type(models.IntegerChoices):
         Creator, on_delete=models.CASCADE, null=True, related_name="notification_recipient", blank=True)
     source = models.ForeignKey(
         Creator, on_delete=models.CASCADE, null=True, related_name="notification_source", blank=True)
+    message = models.CharField(max_length=255, blank=True, null=True)
     link = models.URLField(max_length=1000)
```

Comment:
When I try entering a URL through Django's Admin View like ```/projects/``` it rejects it because it has a built-in URL validator. This isn't an issue when using the endpoint but it would make more sense to use a ```CharField``` because we aren't storing proper URLs anyway. 

See this for reference: https://docs.djangoproject.com/en/dev/ref/models/fields/#urlfield